src/core/css/SearchResult.scss

```diff
@@ -1,13 +1,22 @@
+@import "~core/css/inc/vars";
+
+$border-radius: 9px;
+$secondary-color: #6a6a6a;
+
 .SearchResult {
-  border: solid #cfcfcf;
-  border-width: 1px 1px 0;
+  background: #f5f5f5;
```

Comment:
Are these colours going to be used elsewhere as well? Perhaps they should go into a `amo/css/inc/vars` file?